src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -195,6 +196,33 @@ start_metadata_sync_to_node(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * start_metadata_sync_to_primary_nodes function sets hasmetadata column of
```

Comment:
We should perhaps use a better name like: `resync_metadata_to_primary_nodes` or such?

Reply:
"primary" kind of suggest that you could sync to a different type of node as well. Maybe start_metadata_sync / start_metadata_sync_to_all_nodes?

We'll implicitly replicate to standby nodes.